### ISSUE TITLE
Insert links and Upload Attachment page from Appellate Pacer

### ISSUE THREAD

**Issue by ERosendo** [freelawproject/recap#315]
The extension should insert links on this page and also upload it to the API. This page is similar to the attachment menu from the district pacer. Here's a screenshot of this page:

![image](https://user-images.githubusercontent.com/55959657/205384607-afc8d68e-9fd2-47ff-886b-5cb0cd5237f4.png)

@albertisfu The extension will start sending this page to the API. You can take a look at this page following this link: [attachment page](https://ecf.ca9.uscourts.gov/n/beam/servlet/TransportRoom?servlet=ShowDoc/009031506498)

**Comment by albertisfu** [comment]
Thanks! I've checked and CourtListener can already accept this page.

As we also agreed on slack the fields for submitting this page are:
`court`, `pacer_case_id`, `filepath_local` and the `upload_type`

The upload type is: `APPELLATE_ATTACHMENT_PAGE = 6`

Now we'll only accept and store these pages.

I searched and seems that there's a [PR](https://github.com/freelawproject/juriscraper/pull/358) from @flooie to parse this page in Juriscraper but it's not merged.
